### DISCUSSION TITLE
Does Tailwind throw a warning for non-existent directory in `content` array?

### DISCUSSION THREAD

**Discussion by rakuzen25** [tailwindlabs/tailwindcss#9665] [Help]
Hi! Coming from a Tailwind + SvelteKit setup using [Skeleton](https://github.com/Brain-Bones/skeleton) as my UI library. One thing I realised in Brain-Bones/skeleton#420 is that, under a monorepo with custom libraries, it's very easy for relative paths to be broken due to workspace hoisting configurations.

I've created a reproduction repo for the issue: https://github.com/leranjun/ashes. To give a brief summary, consider the following setup:

```
.
├── node_modules/
│   ├── @brainandbones/
│   │   └── skeleton/
│   │       └── ...
│   └── ...
├── packages/
│   ├── ui/
│   │   ├── tailwind.config.js
│   │   └── ...
│   └── ...
└── ...
```

Tailwind config looks like:
```js
const config = {
	darkMode: 'class',
	content: [
		'./src/**/*.{html,js,svelte,ts}',
		'./node_modules/@brainandbones/skeleton/**/*.{html,js,svelte,ts}' // Problematic!
	],
	plugins: [require('@brainandbones/skeleton/tailwind/theme.cjs')]
};

module.exports = config;
```

Obviously, `node_modules` does not exist at the same level as the config file, so this prevents everything in the library from being loaded and breaks the layout completely.

Interestingly, no warnings showed up - neither in the terminal during dev/build nor in the browser console. Tailwind offers no clue that one of my glob patterns isn't matching anything, and it took me quite a few days to figure out what went wrong.

Is this supposed to be the case? Or is there something about my setup that prevents Tailwind from logging a warning?

**Comment by wongjn** [comment]
Tailwind does not emit warnings for `content` globs that do not exist.

**Reply by rakuzen25** [reply]
I do think it would be much easier for debugging if we can have this feature though, considering relative paths tend to be very fragile with refactors and directory changes. Like linters that report unrecognised/unused imports.

Do I need to open a feature request for this?
